R/derive_param_extreme_record.R

```diff
@@ -136,12 +136,11 @@ derive_param_extreme_record <- function(dataset = NULL,
                                         order,
                                         mode,
                                         set_values_to) {
-  deprecate_inform(
+  deprecate_warn(
     when = "1.2.0",
     what = "derive_param_extreme_record()",
     with = "derive_extreme_event()",
     details = c(
-      x = "This message will turn into a warning at the beginning of 2026.",
```

Comment:
Should we change this to `"This message will turn into an error at the beginning of 2026."`?

Reply:
added back in